### ISSUE TITLE
Windows asking how do you want to open this file, when running task (VSC-199)

### ISSUE THREAD

**Issue by jankrejci** [espressif/vscode-esp-idf-extension#10]
When I run some task, windows asks "How do you want to open this file" (means idf.py).

my workaround is to run python command and put the script as an argument, the example for the Clean task
```
"windows": {
    "command": "python",
    "args": [
        "${config:idf.espIdfPathWin}\\tools\\idf.py",
        "fullclean"
    ]
},
```

Windows 10, VSCode 1.41.1, ESP Extension 0.1.2, ESP-IDF Version 4, Python 3.7.3

**Comment by SoumeshBanerjee** [comment]
Duplicate of #11

**Comment by jankrejci** [comment]
Hi, I think this issue is not duplicate of #11. The #11 is just regarding OpenOCD environment variables. This issue affect all tasks executing some python script. To solve it you need to install python with py launcher and it is needed to check "Assign files with Python" option. Or the second option is to use the workaround above. The issue is not the problem of the extension itself, but I think it should be at least documented or so.

**Comment by SoumeshBanerjee** [comment]
This problem is addressed in #11 and now you should be able to launch from `task.json`, also note that `task.json` can be modify from inside the project to fit your specific environment.
